pkg/api/componentreadiness/triage.go

```diff
@@ -164,6 +164,15 @@ func UpdateTriage(dbc *db.DB, triage models.Triage) (models.Triage, error) {
 		triage.BugID = &bug.ID
 	}
 
+	// When we have removed regressions, we must replace the associations directly, as Save will not handle it.
+	// This results in rows being deleted and re-created even when there are no changes, but this should happen
+	// enough that it makes no difference.
```

Comment:
Total nit -> `but this shouldn't happen
	// enough that it makes a difference.`